tests/generic/001

```diff
@@ -11,7 +11,7 @@ RW=rw
 JOBS=4
 QUEUES=4
 RT=$TRUNTIME
-LOOPS=4
```

Comment:
both generic/001 and generic/002 are hotplug test, which usually requires more loops for triggering issue, 4 is already too small.  But if you care the test time, I'd suggest to figure out one quick test mode by passing 'M=quick' or sort of variable, then all tests can be covered, such as test time, loops or whatever. 

Add one global variable for controlling loops in specific tests looks not good.